RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletGeneratorKernelsImpl.h

```diff
@@ -684,8 +693,9 @@ __global__ void kernel_rejectDuplicate(TrackingRecHit2DSOAView const *__restrict
         auto dop = nSigma2 * (tracks.stateAtBS.covariance(jt)(9) + e2opi);
         if ((opi - opj) * (opi - opj) > dop)
           continue;
-        auto nhj = foundNtuplets.size(jt);
-        if (nhj < nhi || (nhj == nhi && (qj < qi || (qj == qi && score(it, nhi) < score(jt, nhj)))))
+        auto nlj = tracks.nLayers(jt);
+        ;
```

Comment:
```suggestion
```
or clarify why it's needed

Reply:
of course is not needed. thanks for spotting